dev/bot/__main__.py

```diff
@@ -0,0 +1,34 @@
+import asyncio
+import os
+
+import aiohttp
+import discord
+import dotenv
+from discord.ext import commands
+
+import botcore
+from . import Bot
+
+dotenv.load_dotenv()
+botcore.utils.apply_monkey_patches()
+
+roles = os.getenv("ALLOWED_ROLES")
+roles = [int(role) for role in roles.split(",")] if roles else []
+
+bot = Bot(
+    guild_id=int(os.getenv("GUILD_ID")),
```

Comment:
Should we document these required env vars? Same goes for "TOKEN" below.

ALLOWED_ROLES & PREFIX could be documented too, even though optional.

Reply:
Yeah that's a good idea

Reply:
69ae0b0b